docs/form-question-types.rst

```diff
@@ -1068,7 +1068,7 @@ You can specify geometry for all choice sources:
 
 For the first two options, geometry values must be specified in :ref:`the ODK format <location-widgets>`. This makes it straightforward to use data previously collected by ODK as choices displayed on a map. You must make sure that the column containing the geometry to use for each choice has the name ``geometry``.
 
-Learn more about using GeoJSON attachments and see a worked example :ref:`here <selects-from-geojson>`.
+Learn more about using GeoJSON attachments and see a working example :ref:`here <selects-from-geojson>`.
```

Comment:
Worked example: https://tll.mit.edu/teaching-resources/how-people-learn/worked-examples/

If you don’t like it, let’s just go with “example”